src/agent/vsoworker.ts

```diff
@@ -109,7 +109,7 @@ export function run(msg: cm.IWorkerMessage, consoleOutput: boolean,
         trace.state('msg:', msg);
 
         var agentUrl = hostContext.config.settings.serverUrl;
-        var taskUrl = job.environment.variables[cm.vars.systemTfCollectionUri];
+        var taskUrl = job.environment.systemConnection.url;
```

Comment:
in line 37 and 38 of vsoworker, it sets that variable equal to systemConnect.url.  You replaced using the variable with systemConnection.url.  So I'm failing to see how this change made a difference